lib/adapter.js

```diff
@@ -377,7 +377,7 @@ function createMessageListener() {
 function createProbeListener(probeName) {
   monitor.on(probeName, function(res) {
     var duration = 0;
-    if (probeName === 'express') {
+    if (probeName === 'express' && res.response) {
```

Comment:
the intention of the code is that there is only a `res.response.duration` for express probes, I've no reason to think any other probes have one, and I've no idea why the express probe doesn't always have one. This PR is a work-around, we can figure out what is really broken next week - but it doesn't effect apiconnect, which doesn't depend on this metric anyway.